use aoc::y2018::reservoir_research_17::do_solve;
use std::io::Error;

fn main() -> Result<(), Error> {
    aoc::with_input(2018, 17, do_solve)
}
